wsd/RequestDetails.cpp

```diff
@@ -140,17 +170,6 @@ void RequestDetails::processURI()
     _fields[Field::Type] = _uriString.substr(off, posDocUri - off); // The first is always the type.
     std::string uriRes = _uriString.substr(posDocUri + 1);
 
-    // Decode hexified component, if any.
-    if (Util::startsWith(uriRes, "0x"))
```

Comment:
This gave me more confidence - de-hexifying just a specific piece of the URL seems much safer to me during parsing.